ods_ci/tests/Resources/Files/accelerator/qe_create_ap_8.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: dashboard.opendatahub.io/v1
+kind: AcceleratorProfile
+metadata:
+  generation: 1
+  name: qecreateap8
+  namespace: redhat-ods-applications
+spec:
+  description: description example
+  displayName: qe_create_ap_8
```

Comment:
Can you change values with a meaningful name here as well:
- name
- description 
- displayName

Reply:
Hi Fede, the name, description and displayNames fields are yet unclear.
What are "example" and "ap_" ?

Also, we should not use underscore in K8s resources values, please use `-` instead.
See https://kubernetes.io/docs/concepts/overview/working-with-objects/names/
The display name could be as the description I believe, but in the name please split words with `-`.